packages/shared/src/api/dto/UpdateTaskCommand.dto.ts

```diff
@@ -0,0 +1,9 @@
+export interface UpdateTaskCommand {
+  title: string
+  description: string
+  imageCoverId: string
```

Comment:
what about some optional values?

Reply:
description and imageCoverID change to optional values

Reply:
description and imageCoverID change to optional values